src/meta/app/src/schema/table.rs

```diff
@@ -156,6 +156,8 @@ pub struct TableInfo {
 
     pub tenant: String,
 
+    pub database: String,
```

Comment:
I remember there is an assumption that `database` could be changed.

Cc @drmingdrmer for confirmation.